scripts/build.sh

```diff
@@ -204,6 +204,16 @@ case "$product-$method-$platform" in
           test
       fi
 
+      # Run UI tests on both iPad and iphone simultors
```

Comment:
It looks like the IAM tests roughly double the running time of the first build stage.

Given that, IAM should be separated into its own set stage like Firestore is.

Reply:
done with the new commits